crates/runtime/src/traphandlers.rs

```diff
@@ -162,7 +172,7 @@ pub unsafe extern "C" fn wasmtime_call(
     callee: *const VMFunctionBody,
 ) -> Result<(), String> {
     if WasmtimeCall(vmctx as *mut u8, callee) == 0 {
-        Err(trap_message())
+        Err(trap_message_and_stack().0)
```

Comment:
Should this also return `TrapMessageAndStack` instead of `String`?  I think we should display call stack context for trapping start functions.